.editorconfig

```diff
@@ -184,6 +184,9 @@ csharp_space_between_square_brackets = false
 csharp_preserve_single_line_blocks = true
 csharp_preserve_single_line_statements = true
 
+# Async analyzer
+dotnet_diagnostic.CA2016.severity = error
```

Comment:
Duplicated below. Can we remove and have a single definition?

Reply:
This should be the new syntax? `true:error`

Reply:
Let me check. There's the ID ones XXDDDD and there are the named ones...

Reply:
For an ID one or appears to just be the severity: https://github.com/dotnet/roslyn/blob/59644a32be5ad010d87065e8c8238d21f9f6eb5e/.editorconfig#L238